### PR TITLE
Adding PoA FullPage Support

### PR DESCRIPTION
**Pull request by Amit3200**
- Adding PoA FullPage Support
- Support in the options

### CODE REVIEW

File: packages/webdriver-utils/src/providers/automateProvider.js [hunk 1]
```diff
@@ -146,21 +147,17 @@ export default class AutomateProvider extends GenericProvider {
...
         fullscreen,
-        sha: tileData.split('-')[0] // drop build id
+        sha: tileData.sha.split('-')[0] // drop build id
       }));
     }
-
-    const metadata = {
-      windowHeight: Math.ceil(windowHeight * dpr)
-    };
+    const metadata = {};
```
Comment (pankaj443) [inline packages/webdriver-utils/src/providers/automateProvider.js:160]: are we adding anything to it later? can remove if not required

**Review by pankaj443 — APPROVED** [review]
LGTM

File: packages/webdriver-utils/src/providers/automateProvider.js (hunk 1 above)
Comment (rishigupta1599) [reply packages/webdriver-utils/src/providers/automateProvider.js:160]: We are adding cliExecutionTime in it later, which is used for performance metrics.

File: packages/webdriver-utils/src/providers/automateProvider.js [hunk 2]
```diff
@@ -123,15 +123,16 @@ export default class AutomateProvider extends GenericProvider {
     });
   }
 
-  async getTiles(headerHeight, footerHeight, fullscreen) {
+  async getTiles(fullscreen) {
     if (!this.driver) throw new Error('Driver is null, please initialize driver with createDriver().');
     log.debug('Starting actual screenshotting phase');
     const dpr = await this.metaData.devicePixelRatio();
+    this.options.version = 'v2';
```
Comment (itsjwala) [inline packages/webdriver-utils/src/providers/automateProvider.js:130]:
```suggestion
    this.options.version = 'v2';
```

can modification of `options` moved to a single place? [ `percyAutomateRequestHandler` check this function under core/utils ]

Comment (itsjwala) [reply packages/webdriver-utils/src/providers/automateProvider.js:130]: Also spec is pending for same.

**Review by itsjwala — APPROVED** [review]
LGTM

**Review by itsjwala — APPROVED** [review]

**Pull request merged**